src/exoticatechnologies/modifications/ShipModifications.kt

```diff
@@ -214,6 +272,35 @@ class ShipModifications(var bandwidth: Float, var upgrades: ETUpgrades, var exot
         return this.getUpgrade(upgrade) >= upgrade.maxLevel
     }
 
+    /**
+     * Whether we have any exotics or upgrades installed, a convenience method for
+     * ```
+     * hasExotics() || hasUpgrades()
+     * ```
+     *
+     * @return whether we have any exotics or upgrades present or not
+     *
+     * @see hasExotics
+     * @see hasUpgrades
+     */
+    fun hasAnyModifications(): Boolean {
+        return hasExotics() || hasUpgrades()
+    }
+
+    /**
+     * Whether we are under the maximum limit of exotics and can install more, or not
+     *
+     * @param member the [FleetMemberAPI] for which we should check max exotics, used only for determining [FleetMemberAPI.getOwner]
+     *
+     * @see getMaxExotics
+     *
+     * @return whether more exotics can be installed or not
+     */
+    fun isUnderExoticLimit(member: FleetMemberAPI): Boolean {
+        val retVal = getMaxExotics(member) > exotics.getCount(member)
+        return retVal
+    }
```

Comment:
this doesn't look right, does it? shouldn't it be `>=` ? and yet, i know/remember this works fine.